addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.js

```diff
@@ -138,13 +138,28 @@ class ComposerSuggestedRecipient extends Component {
 
     /**
      * @private
+     * @param {object} record the newly-created record
      */
-    _onDialogSaved() {
+    _onDialogSaved(record) {
         const thread = this.suggestedRecipientInfo && this.suggestedRecipientInfo.thread;
         if (!thread) {
             return;
         }
         thread.fetchAndUpdateSuggestedRecipients();
+        if (!this.suggestedRecipientInfo.partner) {
+            this.env.services.notification.notify({
+                title: this.env._t('Invalid Partner'),
+                message: this.env._t('The information you have entered does not match the existing contact information for this record.\
+                             The partner was not created.'),
```

Comment:
Don't forget to add these translatable string in `mail.pot` in stable, so that they are properly registered as text to be translated with transifex.

You can copy how string "Please complete customer's information" is defined in `mail.pot`, as it's defined in the same file.